src/Resources/Resources/RoleAssignments/GetAzureRoleAssignmentCommand.cs

```diff
@@ -245,6 +259,7 @@ public override void ExecuteCmdlet()
                     ResourceType = ResourceType,
                     Subscription = DefaultProfile.DefaultContext.Subscription?.Id?.ToString()
                 },
+                AtScope = AtScope.IsPresent,
```

Comment:
what will be the value if "-AtScope:$false"

Reply:
|                   | `AtScope` | `AtScope.IsPresent` | `this.IsParameterBound(c => c.AtScope)` |
| ----------------- | --------- | ------------------- | --------------------------------------- |
| `-AtScope`        | true      | true                | true                                    |
| (not specified)   | false     | false               | false                                   |
| `-AtScope:$false` | false     | false               | true                                    |

so basically `AtScope` == `AtScope.IsPresent` <> `IsParameterBound()`. Simply use `AtScope` in most cases.